Modules/@babylonjs/react-native/shared/BabylonNative.cpp

```diff
@@ -270,4 +285,22 @@ namespace Babylon
             nativeModule->SetTouchPosition(pointerId, x, y);
         }
     }
+
+    bool IsXRActive()
+    {
+        if (auto nativeModule{ g_nativeModule.lock() })
+        {
+            return nativeModule->IsXRActive();
+        }
+
+        return false;
+    }
+
+    void UpdateXRView(void* windowPtr)
+    {
+        if (auto nativeModule{ g_nativeModule.lock() })
+        {
+            nativeModule->UpdateXRView(windowPtr);
```

Comment:
Is this a no-op on windows? Do we think that any of these changes will make oculus quest (openxr + android) scenarios more difficult to implement?

Reply:
It's basically an "optional" api, in the same way that we used to pass the main windowPtr to the xr.cpp layer, but it was ignored in the OpenXR implementation. So on Windows for now we can just not use this API to pass along a window pointer. Conceptually the difference is between when XR needs to render to some render surface that is managed by the caller vs. a render surface that is managed by xr.cpp itself (such as on an HMD where we are fully immersive and render "full screen"). We probably can do more to rationalize these different scenarios in the NativeXr API, but I think we can do that in a future change.